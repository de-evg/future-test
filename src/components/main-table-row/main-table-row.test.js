import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import MainTableRow from './main-table-row';
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({adapter: new Adapter()});

it(`Render MainTableRow correctly`, () => {
  const wrapper = shallow(<MainTableRow activeRow={"row1"} setActiveRow={() => { }} rowID={"row1"}
    user={{id: 0, firstName: "X", lastName: "X", email: "email@email.ru", phone: 99999999, address: {}, description: "XXX"}} />);
  const tree = shallowToJson(wrapper);
  expect(tree).toMatchSnapshot();
});