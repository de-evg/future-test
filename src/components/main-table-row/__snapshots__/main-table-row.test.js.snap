// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MainTableRow correctly 1`] = `
<Fragment>
  <tr
    className="table__row table__row--active"
    data-active={true}
    onClick={[Function]}
    style={
      Object {
        "backgroudColor": "#fff",
      }
    }
  >
    <TableData
      cellData={0}
      key="cell-0"
    />
    <TableData
      cellData="X"
      key="cell-1"
    />
    <TableData
      cellData="X"
      key="cell-2"
    />
    <TableData
      cellData="email@email.ru"
      key="cell-3"
    />
    <TableData
      cellData={99999999}
      key="cell-4"
    />
  </tr>
  <tr
    className="table__details-row"
  >
    <td
      colspan="5"
    >
      <Details
        details={
          Object {
            "address": Object {},
            "description": "XXX",
            "firstName": "X",
            "lastName": "X",
          }
        }
      />
    </td>
  </tr>
</Fragment>
`;
